src/jira.js

```diff
@@ -300,6 +300,21 @@ export default class JiraApi {
     })));
   }
 
+  /**
+   * @name downloadAttachment
+   * @function
+   * Download an attachment
+   * [Jira Doc](http://docs.atlassian.com/jira/REST/latest/#id288524)
+   * @param {object} attachment - the attachment
+   */
+  downloadAttachment(attachment) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: '/attachment/'.concat(attachment.id).concat('/').concat(attachment.filename),
```

Comment:
I would prefer something like
```js
`/attachment/${attachment.id}/${attachment.filename}`
```
because it's shorter and easier to read. Up to you if you want to update. I'll merge later this week in any case.

Reply:
Thank you for your reply. I just did the modification and commit it.